lint/testdata/lostSQLCloseForRows/positive_tests.go

```diff
@@ -0,0 +1,60 @@
+package checker_tests
+
+import "database/sql"
+
+/// local variable db.Rows have not Close call
+func normalPosLocalUse() {
+	db, err := sql.Open("postgres", "")
+	if err != nil {
+		return
+	}
+
+	rows, err := db.Query("SELECT * FROM testtable")
+	if err != nil {
+		return
+	}
+
+	for rows.Next() {
+		var testdata string
+		rows.Scan(&testdata)
+	}
+
+	return
+}
+
+/// local variable db.Rows have not Close call
+func normalPosUseWhereRowsFromOtherMethod() {
+	db, err := sql.Open("postgres", "")
+	if err != nil {
+		return
+	}
+
+	rows, err := testPosMethodReturningRows(db)
+	if err != nil {
+		return
+	}
+
+	for rows.Next() {
+		var testdata string
+		rows.Scan(&testdata)
+	}
+
+	return
+}
+
+/// param variable db.Rows have not Close call
+func testPosMethodCloseRows(rows *sql.Rows) {
+	for rows.Next() {
+		var testdata string
+		rows.Scan(&testdata)
+	}
+}
+
```

Comment:
Should have `/// param variable db.Rows have not Close call` ?

Reply:
I move it to negative_tests. It should negative, becouse rows returning.